featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -306,8 +306,7 @@ def test_make_agg_feat_of_grandchild_entity(es):
     df = calculator.run(np.array([0]))
     if isinstance(df, dd.DataFrame):
         df = df.compute().set_index('id')
-        df.index = pd.Int64Index(df.index)
-    v = df[agg_feat.get_name()][0]
+    v = df[agg_feat.get_name()].values[0]
```

Comment:
`loc` or `iloc` might work as well

Reply:
Yes, correct. Both of those also work fine. Would you prefer one of those approaches instead?

Reply:
let's use `loc`

Reply:
Updated all to use `.loc` instead of `.values`.